.nvmrc

```diff
@@ -1 +1 @@
-16
+18.15
```

Comment:
You are correct, but in this particular case I pinned it because of ["TypeError: Cannot redefine property: window"](https://github.com/nodejs/node/issues/47563) issue. It was a bug fixed by https://github.com/nodejs/node/pull/46615 but it wasn't back-ported to v18.16 at the time when I made these commits thus causing the aforementioned error, to resolve the issue I pinned it to v18.15. It seems that is has since been back-ported and the issue is now resolved. I have updated the version in `.nvmrc`.